daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto

```diff
@@ -927,6 +927,22 @@ message Expr {
     Expr interface_expr = 3;
   }
 
+  // Upcast from an interface payload to an interface it requires.
+  // *Available in versions >= 1.dev*
+  message ToRequiredInterface {
+    TypeConName required_interface = 1;
+    TypeConName requiring_interface = 2;
+    Expr expr = 3;
```

Comment:
It will be nicer if we were a bit consistent with the name of this kind of nested expression.
For instance, in similar context we use:
- `body`: in `From/ToAny`
- `expr`: in `From/ToAnyException`
- `template_expr` in `From/ToInterface`.

We problably could have make a global pass in the proto and Scala/Haskell Ast to make think a bit more consistent.